students/Deniss_Semcovs/Lesson02/print_grid2.py

```diff
@@ -0,0 +1,8 @@
+ def print_grid(m,n):
+	addl1 = (" -"*(n)+" +")
```

Comment:
Looks like you're using tabs here, make sure you're using spaces or have your text editor convert them for you.  4 spaces per indent.